gluu-server/opt/gluu/jetty/oxauth/custom/scripts/person_authentication/SignInCanada.py

```diff
@@ -455,7 +465,7 @@ def getPageForStep(self, configurationAttributes, step):
                 # The missing file extension below is intentional.
                 # This forces an extra redirect so that the request URL matches a rule in the
```

Comment:
These comments no longer apply and can be removed.

Reply:
Good point Doug. Thanks